src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -88,13 +86,6 @@
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.0.1</PlatformPackageVersion>
   </PropertyGroup>
 
-  <!-- If this package explicitly sets the StableVersion then add it to the stable list -->
```

Comment:
I think we still have some packages that directly set the StableVersion (perhaps only TFS I don't recall), so is there a way to do that still or do they need to be put into the config file?